### DISCUSSION TITLE
When viewing on the Roku app, the "On Now" only shows the network image, not the images shown like in the web ui.

### DISCUSSION THREAD

**Discussion by megahz2** [jellyfin-archive/jellyfin-roku-legacy#1832] [General]
Jellyfin seems to be working fine on the Web UI in windows using SchedulesDirect, I can see images for the "On Now" and the "Details" when clicking an item in the guide. 
However, when viewing on the Roku app, the "On Now" only shows the network image, not the images like shown in the web ui. The same is true for the "details" in the guide 
Has this been fixed? for me Its still an issue and would like...

**Comment by cewert** [comment]
The "On Now" images [were updated](https://github.com/jellyfin/jellyfin-roku/pull/1496) to use show images instead of network images. First appeared in `2.0.4`.

To report a bug please fill out a [bug report](https://github.com/jellyfin/jellyfin-roku/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=) and include as much information as possible
